packer/install-dependencies.sh

```diff
@@ -50,12 +50,7 @@ if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
   overrideNetworkConfig
 fi
 
-ETCD_VERSION="3.3.15"
-ETCD_DOWNLOAD_URL="https://acs-mirror.azureedge.net/github-coreos"
-installEtcd
-echo "  - etcd v${ETCD_VERSION}" >> ${VHD_LOGS_FILEPATH}
-
-MOBY_VERSION="3.0.8"
+MOBY_VERSION="3.0.10"
```

Comment:
This requires changes in installMoby (https://github.com/Azure/aks-engine/pull/2613/files) and it was just adapted by AKS-E master 10 days ago(https://github.com/Azure/aks-engine/pull/2660). I prefer we wait until the next VHD, unless there is fixes required in 3.0.10.